<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetDisplay</class>
 <widget class="QMainWindow" name="PetDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 70, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_user_name">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;b&gt;&lt;font color=red&gt;WELCOME First Name!&lt;font&gt;&lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 175, 177);
border: 2px solid  grey; 
     border-radius: 3px;</string>
      </property>
      <property name="title">
       <string>Picture</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="animalDisplay">
           <property name="minimumSize">
            <size>
             <width>515</width>
             <height>425</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>400</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="text">
              <string>Like</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="text">
              <string>Dislike</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="profileButton">
           <property name="text">
            <string>Check my profile!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 175, 177);
border: 2px solid  grey; 
     border-radius: 3px;</string>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_name">
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_breed">
           <property name="text">
            <string>Breed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_type">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_shelter_name">
           <property name="text">
            <string>Shelter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_location">
           <property name="text">
            <string>Approx location</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 175, 177);
border: 2px solid  grey; 
     border-radius: 3px;</string>
      </property>
      <property name="title">
       <string>Search Attributes</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Type of pet</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Minimum Age</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Breed</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="hypoBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(90, 90, 90);</string>
         </property>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="colorBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(90, 90, 90);</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Minimum Weight</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="hairLenBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(90, 90, 90);</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum Age</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="horizontalSlider_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="breedBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(90, 90, 90);</string>
         </property>
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Affenpinscher</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Curly-Coated Retriever</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Foxhound</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lakeland Terrier</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Hair Length</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="typeBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(90, 90, 90);</string>
         </property>
         <item>
          <property name="text">
           <string>Dog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cat</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Hypoallergenic</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: rgb(176, 175, 177);</string>
         </property>
         <property name="text">
          <string>Maximum Weight</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSlider" name="horizontalSlider_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBread_and_Butter">
    <property name="title">
     <string>Bread and Butter</string>
    </property>
    <addaction name="actionAbout_Bread_and_Butter"/>
    <addaction name="separator"/>
    <addaction name="actionPreferenecs"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionLiked"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGet_Help"/>
   </widget>
   <addaction name="menuBread_and_Butter"/>
   <addaction name="menuProfile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: rgb(176, 175, 177);</string>
   </property>
  </widget>
  <action name="actionAbout_Bread_and_Butter">
   <property name="text">
    <string>About Bread and Butter</string>
   </property>
  </action>
  <action name="actionPreferenecs">
   <property name="text">
    <string>Prefereneces</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionLiked">
   <property name="text">
    <string>Liked</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionGet_Help">
   <property name="text">
    <string>Get Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
